docs/reference/commandline/network_connect.md

```diff
@@ -16,14 +16,38 @@ parent = "smn_cli"
 
       --help=false       Print usage
 
-Connects a running container to a network. This enables instant communication with other containers belonging to the same network.
+Connects a running container to a network. You can connect a container by name
+or by ID. Once connected, the container can communicate with other containers in
+the same network.
 
+```bash
+$ docker network connect multi-host-network container1
 ```
-  $ docker network create -d overlay multi-host-network
-  $ docker run -d --name=container1 busybox top
-  $ docker network connect multi-host-network container1
+
+You can also use the `docker run --net` option to start a container and
+immediately connect it to a network.
+
+```bash
+$ docker run -itd --net=my-bridge-network busybox
 ```
 
-the container will be connected to the network that is created and managed by the driver (multi-host overlay driver in the above example) or external network plugins.
+You can pause, start, and stop containers that are connected to a network. The
+container retains its connection information. When the container is stopped, it
```

Comment:
@mavenugo Do you have any guess under what  conditions these can be lost?  Is it just they can be arbitrarily dropped from the container information or is it something like the network is removed while the container is stopped? Is losing this information more likely in an overlay network and less likely in a bridge network?